<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_setting</class>
 <widget class="QWidget" name="Form_setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,8,2">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>37</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="LineEdit" name="lineEdit_region">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="PushButton" name="btn_register">
         <property name="text">
          <string>注册账号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="ComboBox" name="comboBox_colud_name">
         <item>
          <property name="text">
           <string>腾讯云OSS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>阿里云OSS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="BodyLabel" name="label_4">
         <property name="text">
          <string>地域选择</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="PushButton" name="btn_save_setting">
         <property name="text">
          <string>保存配置</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="BodyLabel" name="label_5">
         <property name="text">
          <string>存储桶名</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="BodyLabel" name="label_2">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>AccessKey ID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="BodyLabel" name="label">
         <property name="text">
          <string>使用云服务商</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="LineEdit" name="lineEdit_bucket_name">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="BodyLabel" name="label_3">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>AccessKey Secret</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="LineEdit" name="lineEdit_key_id">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="3" rowspan="2">
        <widget class="PushButton" name="btn_apply_key">
         <property name="text">
          <string>申请Key</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="LineEdit" name="lineEdit_key_secret"/>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="BodyLabel" name="label_6">
         <property name="text">
          <string>自定义域名</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="LineEdit" name="lineEdit_custom_domain">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>80</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
